website/docs/spec/registry.md

```diff
@@ -152,3 +152,17 @@ The `ros2` profile describes how to create MCAP files for [ROS 2](https://docs.r
 #### Schema
 
 - `encoding`: MUST be either `ros2msg` or `ros2idl`
+
+## Secondary index keys
+
+The Secondary Index Key `name` field may contain the following options:
```

Comment:
Does "may" indicate I can put my own stuff in there and expect some tooling support? I think this would be good to shoot for. Rather than having studio or whatever hard code "header.stamp", "publish_time", etc, would it be viable to dynamically show a list of sort options based on the file's index section?

And likewise with the info command, CLI, reader support etc.

Reply:
Yeah, i feel like you should expect some tooling support for any key, but tooling can make extra assumptions about well-known keys.